identity/backend/src/main/java/io/camunda/identity/usermanagement/Group.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.identity.usermanagement;
+
+public record Group(Integer id, String name) {
```

Comment:
❓ Just wondering, should we use `Integer` here or `UUID`? I personally prefer UUIDs but not sure on the general feeling :)

Reply:
In my idea we can have int for internal/table level relations and use the uuid (as a uk column) whenever we want to make a service available for public or other modules. however here I just used the default ddl of spring security for groups.